### PR TITLE
Add actions to secure notes (view, copy)

### PR DESCRIPTION
**Pull request by jmeridth**
- I miss the ability to copy a note after searching for it, as a former lastpass user.  This hopefully brings that back to me.
- Not too clear on how to use the CrowdIn stuff to translate the rest of the i10n files.  Joined the site and messaged owner.  I think my inquiry below is the process but will wait to see.

### CODE REVIEW

**Comment by jmeridth** [comment]
@kspearrin should I add the new key/values to all of the i10n files and then allow them to be translated by others later?

File: src/_locales/en/messages.json [hunk 1]
```diff
@@ -64,6 +64,9 @@
   "copyPassword": {
     "message": "Copy Password"
   },
+  "copySecurityNote": {
```
Comment (kspearrin) [inline src/_locales/en/messages.json]: Let's just use "Copy Note" to be a bit more generic.

File: src/_locales/en/messages.json [hunk 2]
```diff
@@ -864,6 +867,9 @@
   "securityCode": {
     "message": "Security Code"
   },
+  "secureNote": {
```
Comment (kspearrin) [inline src/_locales/en/messages.json]: Use "Note". Might already exist?

**Comment by kspearrin** [comment]
You only need to provide strings in `en/messages.json`. The others will get generated by crowdin.

**Comment by jmeridth** [comment]
@kspearrin done.  thank you for the feedback.

File: src/_locales/hi/messages.json [hunk 3]
```diff
@@ -1021,4 +1021,4 @@
   "toggleOptions": {
     "message": "Toggle Options"
   }
-}
\ No newline at end of file
+}
```
Comment (kspearrin) [inline src/_locales/hi/messages.json]: can you revert this change still?

**Comment by jmeridth** [comment]
@kspearrin done.  sorry about that. :smile:
